### DISCUSSION TITLE
What to use for XUL addons and JS customizations now that xiaoxiaoflood abandoned this project?

### DISCUSSION THREAD

**Discussion by mzso** [xiaoxiaoflood/firefox-scripts#299] [General Support]
Hi!
I'm still using FF116, because that's where (almost) every customization I had still works.
I use a couple of really useful addons: DTA, Advanced locationbar, Backtrack tab history, Keyconfig, Custom buttons, Dom inspector. Which I don't want to let go since they have no replacements, at most inferior ones.

I also rely heavily on StyloaiX for UI modification. And use old about:config. Plus a handful of other userchrome scripts.

So I wonder what to do now. I can't stay on FF116 forever. I know of some other userscript loaders, though I haven't taken a look at one for years. But I don't remember seeing another XUL extension loader.

**Comment by 117649** [comment]
TMP is hosting a collection of latest loaders :https://onemen.github.io/tabmixplus-docs/other/installation/

**Reply by peterwx** [reply]
It seems the only loader on that page is his ( a single set of instructions is provided ), no mention of other loaders ( accessed the page through Firefox mobile ).

**Reply by 117649** [reply]
what other loaders?

**Comment by peterwx** [comment]
Anyone knows for sure he abandoned this project and the reason, or maybe he's just waiting for the Firefox ESMfication project to conclude or the like, before resuming activity here.

As some of your stuff doesn't work in the most recent versions of Firefox, maybe running mozregression could help you get there .

Aminomancer's repo(s) seems promising ( in terms of quantity and usefulness or quality of customizations  ) although only for CSS and JS  ( user ) scripts, that probably would need conversion of our scripts for its loader ( for simple enough scripts that task should be doable by us, users of this repo or otherwise interested parties ). Legacy addons are presumably left out.

On related ( and not only ) StyloAIX matters, I have to say that I've been working ( and still am ) on an unreleased and unpublished "mod manager" for Firefox Developer Edition and Vivaldi, implemented in python for managing and applying ( among many, many other things, of which, concatenation and minification are a small part) CSS ( and later JS - things like copying and loading or unloading configs, loaders, userscripts, but addons are left out - as I don't use legacy format addons) files ( the "mods" ) located outside the browsers folders to the different browser profiles. It's still very incomplete but I hope to have time to develop, publish and do an initial release closer to the end of the year ( 2024 ) with the basic and fundamental functionality and organized/cleaner/modular source code . It will not include an editor ( as StyloAIX does ), foreseeably.

Also, a few scripts to be polished and contributed here and my own, to be updated, CSS mods for Vivaldi and Firefox, under my account here ( or codeberg ).

**Comment by 117649** [comment]
> Anyone knows for sure he abandoned this project and the reason, or maybe he's just waiting for the Firefox ESMfication project to conclude or the like, before resuming activity here.

No commit to this repo for more then a year now.
